src/components/flow/routers/sheet/helpers.ts

```diff
@@ -50,6 +52,8 @@ export const nodeToState = (settings: NodeEditorSettings): SheetFormState => {
 
       row.value = callSheet.row;
       range.value = callSheet.range;
+      starting_cell.value = callSheet.range.split('!')[1].split(':')[0];
```

Comment:
Since there is no validation for range. This will break if you provide a text that is not in the proper format like:

**Sheet1A1**